src/main/scala/com/yotpo/metorikku/input/readers/mongodb/MongoDBInput.scala

```diff
@@ -82,4 +88,66 @@ object MongoDBInput {
   private def isSingleField(struct : StructType): Boolean = {
     struct.fields.size == 1
   }
+
+  private def sanitizeRow(row : Row): Row = {
+    val sanitizedRow = mutable.ArrayBuffer[Any]()
+
+    for (i <- 0 to row.size - 1) {
+      row.get(i) match {
+        case strValue: String => sanitizedRow += sanitizeObject(strValue)
+        case subRow: Row => {
+          row.schema(i).dataType match {
+            case struct : StructType => {
+              if (isSingleField(struct)) {
+                sanitizedRow += sanitizeObject(subRow.get(0))
+              } else {
+                sanitizedRow += sanitizeRow(subRow)
+              }
+            }
+            case _ => sanitizedRow += sanitizeObject(subRow.get(0))
+          }
+        }
+        case array : Seq[Any] => {
+          if (array.isEmpty) {
+            sanitizedRow += array
+          }
+          else {
+            array(0) match {
+              case _: Row => {
+                val sanitizedSubRowArray = mutable.ArrayBuffer[Any]()
+                array.asInstanceOf[Seq[Row]].foreach(innerRow => {
+                  sanitizedSubRowArray += sanitizeRow(innerRow)
+                })
+                sanitizedRow += sanitizedSubRowArray
+              }
+              case _ => {
+                val sanitizedStringsArray = mutable.ArrayBuffer[Any]()
+                array.foreach(el => {
+                  sanitizedStringsArray += sanitizeObject(el)
+                })
+                sanitizedRow += sanitizedStringsArray
+              }
+            }
+          }
+        }
+        case default =>
+          sanitizedRow += sanitizeObject(default)
+      }
+    }
+    Row.fromSeq(sanitizedRow)
+  }
+
+  private def sanitizeObject(obj : Any): String = {
+    //noinspection ScalaStyle
```

Comment:
why?

Reply:
`null` use, I tried to remove the if but there are cases of null. Without that comment it will fail the ci